### PR TITLE
S1_US2_footer

### PR DESCRIPTION
**Pull request by RaphaelBS-WCS**

### CODE REVIEW

**Review by Vannou28 — COMMENTED** [review]
raph, in your css code, there are too many magic values

**Review by Vannou28 — APPROVED** [review]

**Review by Vannou28 — APPROVED** [review]
Hi raph, I approve but I will have put your phone image url in style.css so that we can use it later.

File: public/assets/css/footer.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    letter-spacing: var(--letter-spacing);
+}
+
+.right-footer {
+    display: flex;
+    flex-direction: column;
+    align-items:center;
+    text-align: center;
+    justify-content: space-around;
+    border-left: 2px solid var(--primary-color);
```
Comment (LucyMerlier) [inline public/assets/css/footer.css:35]: magic value

File: public/assets/css/style.css [hunk 2]
```diff
@@ -1,8 +1,30 @@
...
+
+    --section_width: 100vw;
+    --width-Big: 90%;
+    --width-Full: 100%;
+    --width-Middle: 40%;
+    --width-half: 50%;
+    --phone-width:200px;
+    --resp-phone-width:130px;
+    --phone-height:33px;
+    --height:80%;
```
Comment (LucyMerlier) [inline public/assets/css/style.css:29]: incoherent variable names : use kebab-case

File: src/View/component/_footer.html.twig [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+<section class="nath-footer">
+    <div class="left-footer">
+        NATHALIE MILLIET
+    </div>
+    <div class="right-footer">
+        <div class="footer-contact">
+            <a href="#" alt="contact">CONTACT</a>
+        </div>
+        <div class="footer-phone">
+        </div>
```
Comment (LucyMerlier) [inline src/View/component/_footer.html.twig:14]: empty div

File: src/View/component/_footer.html.twig [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    <div class="right-footer">
+        <div class="footer-contact">
+            <a href="#" alt="contact">CONTACT</a>
+        </div>
+        <div class="footer-phone">
+        </div>    
+    </div>
+</section>
+
+{% endblock %}
```
Comment (LucyMerlier) [inline src/View/component/_footer.html.twig:18]: this is a component, only made to be included, so it should not contain twig blocks

File: src/View/layout.html.twig [hunk 5]
```diff
@@ -7,17 +7,19 @@
...
     <link rel="icon" href="/assets/images/favicon.png">
 </head>
 
 <body>
+    {% block navbar %}
+    {% endblock %}
     {% block content %}{% endblock %}
+    {% block footer %}
+         {% include 'component/_footer.html.twig' %}
+    {% endblock %}
```
Comment (LucyMerlier) [inline src/View/layout.html.twig:21]: including block footer in block footer

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: src/View/component/_footer.html.twig (hunk 3 above)
Comment (RaphaelBS-WCS) [reply src/View/component/_footer.html.twig:12]: it was filled by the css background containing the phone image.

File: public/assets/css/style.css (hunk 2 above)
Comment (RaphaelBS-WCS) [reply public/assets/css/style.css]: alright we correct it

File: src/View/component/_footer.html.twig (hunk 4 above)
Comment (RaphaelBS-WCS) [reply src/View/component/_footer.html.twig]: i've correct it

File: src/View/layout.html.twig (hunk 5 above)
Comment (RaphaelBS-WCS) [reply src/View/layout.html.twig]: i've correct it

**Review by Mcbess-san — APPROVED** [review]

File: src/View/component/_footer.html.twig [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+        </div>
+        <div class="col-xs-12 col-sm-6 text-center" id="footer_right">
+            <div class="row">
+                <a class="text-reset text-decoration-none fs-3"href="#" alt="contact">CONTACT</a></div>
+            <div class="row">
+                <div class="col-xs-12 col-sm-6-ml-3">
+                    <img src='assets/images/telephone.png' alt='telephone'></div>
+                </div>
+        </div>
+    </div>
```
Comment (LucyMerlier) [inline src/View/component/_footer.html.twig:13]: indentation

File: src/View/component/_footer.html.twig [hunk 7]
```diff
@@ -0,0 +1,13 @@
+    <div class="row align-items-center fw-bold" id="footer">
+        <div class="col-xs-12 col-sm-6 text-center fs-1" id="footer_left">
+        <p>NATHALIE MILLIET</p>
```
Comment (LucyMerlier) [inline src/View/component/_footer.html.twig:3]: couldn't this be a title?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: src/View/component/_footer.html.twig (hunk 7 above)
Comment (RaphaelBS-WCS) [reply src/View/component/_footer.html.twig:3]: yeah, it could and it will. It's done!

File: src/View/component/_footer.html.twig (hunk 6 above)
Comment (RaphaelBS-WCS) [reply src/View/component/_footer.html.twig:13]: OMG yes Sorry

**Review by Mcbess-san — APPROVED** [review]

File: src/View/component/_footer.html.twig [hunk 8]
```diff
@@ -0,0 +1,13 @@
+<div class="row align-items-center fw-bold" id="footer">
+    <div class="col-xs-12 col-sm-6 text-center fs-1" id="footer_left">
+        <h1>NATHALIE MILLIET</h1>
```
Comment (LucyMerlier) [inline src/View/component/_footer.html.twig:3]: so this will be the only h1 on the entire page?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

File: src/View/component/_footer.html.twig (hunk 8 above)
Comment (RaphaelBS-WCS) [reply src/View/component/_footer.html.twig:3]: yes, we think so... is this a problem?

Comment (RaphaelBS-WCS) [reply src/View/component/_footer.html.twig:3]: i'll transform this in a h2 after adding h1 to navbar

**Review by Vannou28 — APPROVED** [review]
it's good !!!

**Review by LucyMerlier — DISMISSED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Review by Erdnaxela45 — APPROVED** [review]
wow

**Pull request merged**